config/apiserver/rbac/user_clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: bulward:user
```

Comment:
Also for this name, `bulward:` can be dropped.

Reply:
same as above